Scripts/Tracking/Velocity/VelocityEstimator.cs

```diff
@@ -39,19 +39,7 @@ public class VelocityEstimator : VelocityTracker
         /// <inheritdoc />
         public override bool IsActive()
         {
-            return (isActiveAndEnabled && source != null && source.gameObject.activeInHierarchy);
-        }
-
-        /// <inheritdoc />
-        public override Vector3 GetVelocity()
-        {
-            return GetEstimate(velocitySamples);
-        }
-
-        /// <inheritdoc />
-        public override Vector3 GetAngularVelocity()
-        {
-            return GetEstimate(angularVelocitySamples);
+            return (isActiveAndEnabled && source != null && source.gameObject.activeInHierarchy && isActiveAndEnabled);

```

Comment:
Duplicated `isActiveAndEnabled`. Should this just be
```c#
return (isActiveAndEnabled && source != null && source.isActiveAndEnabled);
```
perhaps?

Reply:
source is a transform so has no access to isActiveAndEnabled, but the rest is legit